modules/account-lib/src/coin/algo/keyPair.ts

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-unused-vars */
+import { PrivateKey, PublicKey } from '@hashgraph/sdk';
```

Comment:
We shouldn't be using the hedera hashgraph library for algorand. Algorand doesnt have a special format so this can be a simple passthrough implementation. 

i.e. `return { pub }`